pymeasure/instruments/keithley/keithley6221.py

```diff
@@ -145,6 +155,117 @@ def __init__(self, adapter, name="Keithley 6221 SourceMeter", **kwargs):
         map_values=True,
     )
 
+    ##############
+    # DELTA MODE #
+    ##############
+
+    delta_unit = Instrument.control(
+        ":UNIT:VOLT:DC?", ":UNIT:VOLT:DC %s",
+        """ A string property that controls the reading unit.
+        Valid values are 'V', 'Ohms', 'W' and 'Siemens' """,
+        validator=strict_discrete_set,
+        values={"V": "V", "Ohms": "OHMS", "W": "W", "Siemens": "SIEM"},
+        map_values=True
+    )
+
+    delta_high_source = Instrument.control(
+        ":SOUR:DELT:HIGH?", ":SOUR:DELT:HIGH %g",
+        """ A floating point property that controls the delta high source value in Amps. """,
+        validator=truncated_range,
+        values=[0, 0.105]
+    )
+
+    delta_low_source = Instrument.control(
+        ":SOUR:DELT:LOW?", ":SOUR:DELT:LOW %g",
+        """ A floating point property that controls the delta
+        low source value in Amps. By default,
+        the low source value is minus the high source value.""",
+        validator=truncated_range,
+        values=[-0.105, 0]
+    )
+
+    delta_delay = Instrument.control(
+        ":SOUR:DELT:DELay?", ":SOUR:DELT:DELay %s",
+        """ A floating point property that controls the delta delay
+        in seconds. Can be a value between 0 and 9999.999,
+        or "INF" for infinite delay.""",
+        validator=joined_validators(truncated_range, strict_discrete_set),
+        values=([0, 9999.999], ["INF"]),
+    )
+
+    delta_cycles = Instrument.control(
+        ":SOUR:DELT:COUN?", ":SOUR:DELT:COUN %s",
+        """ An integer property that controls the number of cycles
+        to run for the delta measurements. Can be a value
+        between 1 and 65536, or "INF" for infinite cycles.""",
+        validator=joined_validators(truncated_range, strict_discrete_set),
+        values=([1, 65536], ["INF"]),
+    )
+
+    delta_measurement_sets = Instrument.control(
+        ":SOUR:SWEep:COUN?", ":SOUR:SWEep:COUN %s",
+        """ An integer property that controls the number of
+        measurement sets to repeat for delta measurements. Can be
+        a value between 1 and 65536, or "INF" for infinite sets.""",
+        validator=joined_validators(truncated_range, strict_discrete_set),
+        values=([1, 65536], ["INF"]),
+    )
+
+    delta_compliance_abort = Instrument.control(
+        ":SOUR:DELT:CAB?", ":SOUR:DELT:CAB %s",
+        """ A boolean property that controls whether the
+        compliance abort is turned on or off.
+        Valid values True (on) or False (off). """,
+        values={True: "ON", False: "OFF"},
+        map_values=True,
+    )
+
+    delta_cold_switch = Instrument.control(
```

Comment:
For boolean properties, names like `delta_cold_switch_enabled` are self-explanatory.